includes/admin/class-wc-admin-addons.php

```diff
@@ -126,6 +149,214 @@ public static function output_storefront_button() {
+			<?php
+		} else {
+			?>
+			</div>
+			<?php
+		}
+		if ( isset( $block->container ) && $block->container === 'column_container_end' ) {
+			?>
+			</div>
+			<?php
+		}
+	}
+
+	/**
+	 * Handles the outputting of a column block.
+	 *
+	 * @param object $block
+	 */
+	public static function output_column_block( $block ) {
+		?>
+		<div class="addons-column-block">
+			<h1><?php echo esc_html( $block->title ); ?></h1>
+			<p><?php echo esc_html( $block->description ); ?></p>
+			<?php foreach ( $block->items as $item ) : ?>
+				<div class="addons-column-block-item">
+					<div class="addons-column-block-item-icon">
+						<img class="addons-img" src="<?php echo esc_attr( $item->image ); ?>" />
+					</div>
+
+					<div class="addons-column-block-item-content">
+						<h2><?php echo esc_html( $item->title ); ?></h2>
+						<?php
+							self::output_button(
+								$item->href,
+								$item->button,
+								'addons-button-solid',
+								$item->plugin
+							);
+						?>
+						<p><?php echo esc_html( $item->description ); ?></p>
+
+					</div>
+				</div>
+			<?php endforeach; ?>
+		</div>
+
+		<?php
+	}
+
+	/**
+	 * Handles the outputting of a small light block.
+	 *
+	 * @param object $block
+	 */
+	public static function output_small_light_block( $block ) {
+		?>
+		<div class="addons-small-light-block">
+			<img class="addons-img" src="<?php echo esc_attr( $block->image ) ?>" />
+			<div class="addons-small-light-block-content">
+				<h1><?php echo esc_html( $block->title ); ?></h1>
+				<p><?php echo esc_html( $block->description ); ?></p>
+				<div class="addons-small-light-block-buttons">
+					<?php foreach ( $block->buttons as $button ) : ?>
+						<?php
+							self::output_button(
+								$button->href,
+								$button->text,
+								'addons-button-solid'
+							);
+						?>
+					<?php endforeach; ?>
+				</div>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Handles the outputting of a small dark block.
+	 *
+	 * @param object $block
+	 */
+	public static function output_small_dark_block( $block ) {
+		?>
+		<div class="addons-small-dark-block">
+			<h1><?php echo esc_html( $block->title ); ?></h1>
+			<p><?php echo esc_html( $block->description ); ?></p>
+			<div class="addons-small-dark-items">
+				<?php foreach ( $block->items as $item ) : ?>
+					<div class="addons-small-dark-item">
+						<div class="addons-small-dark-item-icon">
+							<img class="addons-img" src="<?php echo esc_attr( $item->image ); ?>" />
+						</div>
+						<?php
+							self::output_button(
+								$item->href,
+								$item->button,
+								'addons-button-outline-white'
+							);
+						?>
+					</div>
+				<?php endforeach; ?>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Handles the outputting of featured sections
+	 *
+	 * @param array $sections
+	 */
+	public static function output_featured_sections( $sections ) {
+		foreach ( $sections as $section ) {
+			switch ($section->module) {
+				case "banner_block":
```

Comment:
Never user double quotes if you don't need double quotes.
Need to change in all occurrences inside this method.